src/sentry/testutils/pytest/fixtures.py

```diff
@@ -361,10 +363,12 @@ def inner(
             if subname is not None:
                 name += f"_{subname}"
 
+            parent_name = strip_silo_mode_test_suffix(request.node.parent.name)
```

Comment:
Hard-coding this directly in `insta_snapshot` implies that we always expect the same snapshot to reflect expected behavior in both monolith mode and the siloed mode. Generally, I'm for that, because behavioral parity between modes is a good thing.

By contrast, if we *don't* expect behavioral parity, it might be desirable to have (for example) one file named `tests/sentry/tasks/snapshots/PreprocessingCompleteTest/test_success.pysnap` and another named `tests/sentry/tasks/snapshots/PreprocessingCompleteTest__InRegionMode/test_success.pysnap`. (Which is what this function was expecting before the changes in this PR, with the snag that the latter file doesn't exist.)

I figure, if we ever do need to do it the second way, we could easily add an optional argument for it to `insta_snapshot`.

Reply:
Maybe worth leaving this as a (shorter) comment above this line in the code for future readers?